mcs/class/System/Test/System.Net.Security/SslStreamTest.cs

```diff
@@ -93,6 +171,167 @@ void AuthenticateClientAndServer (bool server, bool client)
 		}
 	}
 
+	[Category("NotWorking")]    // Works in .Net, doesn't work in mono.
+	[Test()]
+	public void SelfCompatibilityTest()
+	{
+		#if __MonoCS__
```

Comment:
If you just don't like using ```#if __MonoCS__```, another perfectly acceptable alternative would be to use a specifically purposed compiler symbol that exists only in the windows project.  For example, instead of

    #if __MonoCS__
    DoMonoStuff();
    #else
    DoDotNetStuff();
    #endif

It would be fine to do this:

    #if ! DOTNET
    DoMonoStuff();
    #else
    DoDotNetStuff();
    #endif